### PR TITLE
[Android] NativeScrollViewerPresenter was clipping its children.

### PR DESCRIPTION
**Pull request by carldebilly**
Fixes https://github.com/unoplatform/nventive-private/issues/31
# Bugfix

[Android] `NativeScrollViewerPresenter` were clipping its children. This is the role of the `ScrollViewer` itself, not the presenter.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [X] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/NativeScrollContentPresenter.Android.cs [hunk 1]
```diff
@@ -55,6 +55,7 @@ public NativeScrollContentPresenter()
 			SetForegroundGravity(GravityFlags.Fill);
 
 			SetClipToPadding(false);
+			SetClipChildren(false);
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/NativeScrollContentPresenter.Android.cs:58]: What does this change? Is it possible to add a test for it?

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/NativeScrollContentPresenter.Android.cs:58]: Indeed, automated tests to validate this are needed.

**Comment by jeromelaban** [comment]
Build failure:
```
##[error]src\SamplesApp\UITests.Shared\Toolkit\ElevatedView_Corners.xaml(14,73): Error WMC0011: Unknown member 'Weight' on element 'TextBlock'
```

**Comment by jeromelaban** [comment]
This does not look right: 

https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=21078&view=ms.vss-test-web.build-test-results-tab&runId=1200966&resultId=100414&paneView=attachments

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/beta/3.1

**Pull request merged**